### PR TITLE
MEGAHIT Tool Wrapper Update

### PR DESCRIPTION
**Pull request by jraysajulga**
Looking to enable the k-list parameter and intermediate_contigs output for use as assembly graphs. Also, added in help documentation referenced from the [GitHub](https://github.com/voutcn/megahit/wiki/Assembly-Tips) documentation on assembly tips.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/megahit/megahit_wrapper.xml [hunk 1]
```diff
@@ -22,15 +22,20 @@ megahit
...
         -r '${input_option.single_files}'
     #end if
     ##basic assembly
     --min-count '${basic_section.min_count}'
-    --k-min '${basic_section.k_min}'
-    --k-max '${basic_section.k_max}'
-    --k-step '${basic_section.k_step}'
-    --min-contig-len '${basic_section.min_contig_len}'
+    #if $basic_section["k_mer"].k_mer_method == "klist_method":
+        --k-list '${basic_section["k_mer"].k_list}'
```
Comment (abretaud) [inline tools/megahit/megahit_wrapper.xml]:
```suggestion
        --k-list '${basic_section.k_mer.k_list}'
```
And the same for the few following lines

**Comment by jraysajulga** [comment]
@abretaud 

> Cool, thanks for the PR, just a little comment

Thanks. I was wondering about this... Looks a lot better now.

**Comment by abretaud** [comment]
Thanks @jraysajulga!

**Comment by jraysajulga** [comment]
No problem. Thanks for the quick review!

**Pull request merged**